src/extension/interop/ExtensionInterop.h

```diff
@@ -6,6 +6,13 @@
 
 #include "interop/CStructures.h"
 
+#include "atlas/atlas_unwrap.hpp"
+
+#include <godot_cpp/classes/mesh.hpp>
+#include <godot_cpp/classes/array_mesh.hpp>
+
```

Comment:
This file is meant to be pure C code, so this is wrong to include these files here at all.